src/sentry/workflow_engine/models/action_group_status.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import DefaultFieldsModel, FlexibleForeignKey, region_silo_model
+
+
+@region_silo_model
+class ActionGroupStatus(DefaultFieldsModel):
+    """
+    Stores when a workflow last fired for a Group.
+    """
+
+    ACTIVE = 0
+    INACTIVE = 1
+
+    __relocation_scope__ = RelocationScope.Excluded
+
+    action = FlexibleForeignKey("workflow_engine.Action", on_delete=models.CASCADE)
```

Comment:
We should probably add this to `deletions/` - one action could potentially be associated with many groups, and so this could be a large deletion. Feel free to make that a separate pr